### ISSUE TITLE
Requirements not specified

### ISSUE THREAD

**Issue by hno2lr-xcdx** [onion-liu/BlendGAN#16]
The readme contains information how to run the python scripts and the notebook.
- for installing the scripts: Which requirements are needed?
- hint: [this Pull ](https://github.com/onion-liu/BlendGAN/pull/2)request is hinting to some requirements
- Hardware/pypi packages?

I ran the scripts on a laptop with 4GB GPU size.
But:
- only the [generate_image_pairs](https://github.com/onion-liu/BlendGAN/blob/main/generate_image_pairs.py) is working, the other script [style_transfer_folder](https://github.com/onion-liu/BlendGAN/blob/main/style_transfer_folder.py) is throwing a CUDA OutOfMemoryError.
- jupyter notebook is throwing the same CUDA error.

Thanks in advance for a short  feedback.

**Comment by onion-liu** [comment]
The script [style_transfer_folder.py](https://github.com/onion-liu/BlendGAN/blob/main/style_transfer_folder.py) requires more GPU memory than [generate_image_pairs.py](https://github.com/onion-liu/BlendGAN/blob/main/generate_image_pairs.py) because it loads another model (PSPEncoder). Perhaps you need another GPU with more memory to run style_transfer_folder.py

You can also split style_transfer_folder.py into two steps: 
Step 1: calculate [sample_in](https://github.com/onion-liu/BlendGAN/blob/8da1ea199cd4bf0c0dede4198f85034d84db69bb/style_transfer_folder.py#L83) with psp_encoder and save sample_in into a .npy file. 
Step 2: generate [img_out_ten](https://github.com/onion-liu/BlendGAN/blob/8da1ea199cd4bf0c0dede4198f85034d84db69bb/style_transfer_folder.py#L84) from sample_in with the model g_ema to get the final result.

**Comment by ghost** [comment]
Ok, thanks for the feedback.

Is it possible to mitigate the problem by using a smaller version of the PSPEncoder model?

**Comment by onion-liu** [comment]
> 

Yes. But you need to train the new version of the PSPEncoder by yourself.
